jwst/assign_wcs/nirspec.py

```diff
@@ -634,8 +634,8 @@ def get_open_msa_slits(msa_file, msa_metadata_id, dither_position,
             else:
                 jmin = min([s['shutter_column'] for s in slitlets_sid])
                 jmax = max([s['shutter_column'] for s in slitlets_sid])
-                j = jmin + (jmax - jmin) // 2 + 1
-            ymax = 0.5 + margin + (jmax - j) * 1.15
+                j = jmin + (jmax - jmin) // 2
```

Comment:
I finally convinced myself that this change is correct. It is important that `j` and `ycen` are 1-based and with this change they are because the msa metadata file `shutter_column` values are 1-based. They need to be 1-based in order to compute the correct [shutter_id](https://github.com/spacetelescope/jwst/pull/6454/files#diff-45fd9f060b3edd128f0e133edcd2124e8ae1b90140b72adffe3a4e2f715049dfL673). 
The extra `+1` previously may have been based on an assumption that `jmin` and `jmax` are 0-based.

Reply:
And it does seem to work correctly now too, based on actual data coming out of the step. In the past, if a background slit contained 3 shutters in columns 21, 22, 23 (for example), it set j=ycen=23 and the shutter_state string was "11x". Both results indicating the source location in an end shutter. Now it gives j=ycen=22 and "1x1" (source in the middle).